libs/langchain-cohere/src/chat_models.ts

```diff
@@ -189,40 +273,256 @@ export class ChatCohere<
       searchQueriesOnly: options.searchQueriesOnly,
       documents: options.documents,
       temperature: options.temperature ?? this.temperature,
+      forceSingleStep: options.forceSingleStep,
+      tools: options.tools,
     };
     // Filter undefined entries
     return Object.fromEntries(
       Object.entries(params).filter(([, value]) => value !== undefined)
     );
   }
 
+  /** @ignore */
```

Comment:
Make the method `private` or `protected` and prefix with an underscore (`_`) instead